### ISSUE TITLE
"start a fire quickly" should use the quickest object possible

### ISSUE THREAD

**Issue by BaumgeistOne** [CleverRaven/Cataclysm-DDA#53203]
### Is your feature request related to a problem? Please describe.

I am a big fan of the action menu, and I am a big fan of burning zombies (and fungus), so naturally I use the "start a fire quickly" action quite a lot. Works great while it is nighttime, but during the day, while having binoculars in the inventory, the character uses those to light the fire, instead of one of the dozens of lighters I always collect and keep around. I get that this saves a charge of the lighter, but it wastes precious time, and frankly time is most often the more valuable resource.

### Solution you would like.

"start a fire quickly" should consider the time an item takes to ignite the fire, and the action should use the quickest item possible.

### Describe alternatives you have considered.

A new action "start a fire slowly" (using binoculars and similar items) would solve the problem.

Simply removing the binoculars from the "start a fire quickly" action would solve the problem too.

**Comment by ChainsawGarden** [comment]
Agreed. If you are going to start a fire quickly, it's naturally fitting that you use the object that will cost you the least amount of turns to pick up and use.
